Makefile

```diff
@@ -46,3 +45,38 @@ python-bootstrap: venv
 
 venv:
 	test -d venv || virtualenv venv
+
+memorydiskserver-docker-image:
+	mkdir -p memorydiskserver-build
+	CGO_ENABLED=0 GOOS=linux godep go build -a -ldflags '-s' cmd/memorydiskserver/memorydiskserver.go
+	mv memorydiskserver memorydiskserver-build/
+	cp Dockerfile.memorydiskserver memorydiskserver-build/Dockerfile
+	cd memorydiskserver-build && docker build -t clusterhq/memorydiskserver .
+
+dvol-golang-docker-image:
```

Comment:
I would prefer this target was a script in the scripts folder and the makefile calls the script.